tofu/geom/utils.py

```diff
@@ -695,8 +700,40 @@ def _compute_CamLOS2D_pinhole(P=None, F=0.1, D12=0.1, N12=100,
                   'B4': 'ITER-V2',
                   'NSTX': 'NSTX-V0'}
 
-# Default config
-_DEFCONFIG = 'ITER'
+
+def _get_listconfig(dconfig=_DCONFIG, dconfig_table=_DCONFIG_TABLE,
+                    returnas=str):
+    assert returnas in [dict, str]
+    dc = {k0: [k0] + sorted([k1 for k1, v1 in dconfig_table.items()
+                             if v1 == k0])
+          for k0 in sorted(dconfig.keys())}
+    if returnas is dict:
+        return dc
+    else:
+        l0 = np.max([len(k0) for k0 in dc.keys()] + [len('unique names')])
+        l1 = np.max([len(str(v0)) for v0 in dc.values()] + [len('shortcuts')])
+        msg = ("\n\t" + "unique names".ljust(l0) + "\tshortcuts"
+               + "\n\t" + "-"*l0 + " \t" + "-"*l1
+               + "\n\t- "
+               + "\n\t- ".join(["{}\t{}".format(k0.ljust(l0), v0)
+                                for k0, v0 in dc.items()]))
+        return msg
+
+
+def get_available_config(dconfig=_DCONFIG, dconfig_table=_DCONFIG_TABLE,
```

Comment:
Public method for end-users

Reply:
Same here, a docstring would be nice.

Also, since this is a new public method, it should be added to `__all__` no? As far as I can tell this is not the case.

Otherwise the message is nicely written, especially the use of an URL is good I believe.

Reply:
Done, thanks :-)